tests/Temporalio.Tests/TestUtils.cs

```diff
@@ -224,4 +225,60 @@ public record CaptureMetric<T>(string Name, string? Unit, string? Description)
 
         public void Set(T value, object tags) => Values.Enqueue((value, (Dictionary<string, object>)tags));
     }
-}
\ No newline at end of file
+
+    public static async Task<Temporalio.Api.WorkflowService.V1.SetWorkerDeploymentRampingVersionResponse>
+    SetRampingVersionAsync(
+        ITemporalClient client,
+        Google.Protobuf.ByteString conflictToken,
+        WorkerDeploymentVersion version,
+        int rampPercentage)
+    {
+        return await client.WorkflowService.SetWorkerDeploymentRampingVersionAsync(
+            new()
+            {
+                Namespace = client.Options.Namespace,
+                DeploymentName = version.DeploymentName,
+                Version = version.ToCanonicalString(),
+                ConflictToken = conflictToken,
+                Percentage = rampPercentage,
+            });
+    }
+
+    public static async Task<Temporalio.Api.WorkflowService.V1.DescribeWorkerDeploymentResponse>
```

Comment:
I'd personally toss this in `AssertMore` and call it `EventuallyWorkerDeploymentPresentAsync` or something, but no biggie